### PR TITLE
Improve the Shape/Shape2D classes documentation

### PR DESCRIPTION
**Pull request by Calinou**
This makes the Shape/Shape2D classes 100% documented.

### CODE REVIEW

File: doc/classes/Physics2DShapeQueryResult.xml [hunk 1]
```diff
@@ -27,6 +31,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns the ID of the [Object] that intersected with the shape at index [code]idx[/code].
```
Comment (Calinou) [inline doc/classes/Physics2DShapeQueryResult.xml]: Is the object ID the same as the one returned by `Object.get_instance_id()`? I haven't had the time to check this.

Comment (akien-mga) [reply doc/classes/Physics2DShapeQueryResult.xml]: Seems so:
```
servers/physics_2d/space_2d_sw.cpp
93:             r_results[cc].collider_id = col_obj->get_instance_id();
94:             if (r_results[cc].collider_id != 0)
95:                     r_results[cc].collider = ObjectDB::get_instance(r_results[cc].collider_id);
184:    r_result.collider_id = res_obj->get_instance_id();
185:    if (r_result.collider_id != 0)
186:            r_result.collider = ObjectDB::get_instance(r_result.collider_id);
```

File: doc/classes/Shape2D.xml [hunk 2]
```diff
@@ -77,6 +77,7 @@
 	</methods>
 	<members>
 		<member name="custom_solver_bias" type="float" setter="set_custom_solver_bias" getter="get_custom_solver_bias" default="0.0">
+			The shape's custom solvier bias.
```
Comment (akien-mga) [inline doc/classes/Shape2D.xml]: Typo, "solver".

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**